faf-commons-lobby/src/main/kotlin/com/faforever/commons/lobby/GameApi.kt

```diff
@@ -179,6 +179,8 @@ data class GameLaunchResponse(
   val expectedPlayers: Int? = null,
   @JsonProperty("map_position")
   val mapPosition: Int? = null,
+  @JsonProperty("game_options")
+  val gameOptions: Map<String, String>? = null,
```

Comment:
In the example of the PR you referenced, the values are not necessarily Strings?

Reply:
I know numbers get converted properly to strings but do booleans have the correct casing we need when converted?

Reply:
Tested it, Jackson can convert everything to String